Presentation.Web/Controllers/API/V1/OrganizationController.cs

```diff
@@ -37,39 +42,24 @@ public virtual HttpResponseMessage Get([FromUri] string q, [FromUri] PagingModel
             return GetAll(paging);
         }
 
-        public HttpResponseMessage GetBySearch(string q, int orgId, int take = 25)
+        public HttpResponseMessage GetBySearchAndOrganizationId(string q, bool onlyWhereUserHasMembership, [FromUri] V1BoundedPaginationQuery pagination = null)
```

Comment:
I don't understand why the `onlyWhereUserHasMembership`.. has been added to this endpoint. It is never to be true for the use cases that use this endpoint (in v2 it replaces the getauthentication endpoint) 

So kill that